test/run_pass/test_require_resolve.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+var assert = require('assert');
+var fixtures = require('../common/fixtures');
+
+assert.strictEqual(
+  require.resolve('test_util').toLowerCase(),
```

Comment:
Requiring relative modules without leading `./` are not guaranteed on API stability. Tests could cover cases with leading `./` instead.

Reply:
Yes, I am negligent.